### PR TITLE
Remove 3dtiles layer by ID

### PR DESCRIPTION
**Pull request by clementcolin**

### CODE REVIEW

File: packages/browser/src/Component/Itowns/LayerManager/LayerManager.js [hunk 1]
```diff
@@ -255,6 +255,21 @@ export class LayerManager {
     return undefined;
   }
 
+  /**
+   * Remove a a layer and its tilesManager given a layer ID.
+   *
+   * @param {string} id the layer ID.
+   * @returns {TilesManager}
```
Comment (valentinMachado) [inline packages/browser/src/Component/Itowns/LayerManager/LayerManager.js:262]: this function return nothing

**Review by valentinMachado — CHANGES_REQUESTED** [review]
Same comment as https://github.com/VCityTeam/UD-Viz/pull/542

**Comment by clementcolin** [comment]
Should be good now :)

**Comment by valentinMachado** [comment]
eslint doesnt pass in index.js ^^ but everything seems fine to me

**Pull request merged**